deployment/minikube/generate_yaml.sh

```diff
@@ -38,5 +38,6 @@ yq w -i --style=double --inplace -- deployment.yaml spec.template.spec.container
 
 # ESP service configuration
 yq w --style=double ../template/endpoints.yaml.tmpl name $SERVICE_NAME > endpoints.yaml
+yq w -i endpoints.yaml title $service_name
 
 # Deploy
```

Comment:
Is there a missing step after this comment? Or does another script take over after?

Reply:
Removed as this is stale. Another script will take over.